### PR TITLE
added section for git installation on Chrome

### LINKED ISSUE #12012: Add clear instructions to use git on ChromeOS devices
### Code of Conduct

- [x] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/get-started/quickstart/set-up-git

### What part(s) of the article would you like to see updated?

Give a specific set of instructions on how to set up git on a ChromeOS device (because you can't install GitHub for desktop on ChromeOS devices). Many people, especially high school students, have Chromebooks, so it would be helpful to have clear documentation on how to set up Github rather than them having to look for info on how to do it online, which is a lot harder. Maybe write something about a terminal-like app you can download in order to install git, like this article: https://www.techrepublic.com/article/how-to-use-github-in-chrome-os/. This would help people easily get started with coding if they have a ChromeOS device. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by nayasinghania**
### Why:

Closes https://github.com/github/docs/issues/12012

### What's being changed:
Added 3 lines of instructions under 1. download and install git for Chrome OS users to be able to install git on their devices

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@vedant205 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by HLA95** [comment]
tests/content/featured-links.js

**Comment by HLA95** [comment]
Add

File: content/get-started/quickstart/set-up-git.md
Comment (skedwards88) [inline content/get-started/quickstart/set-up-git.md:45]:
```suggestion

{% note %}

**Note**: If you are using a Chrome OS device, additional set up is required:

1. Install a terminal emulator such as Termux from the Google Play Store on your Chrome OS device.
2. From the terminal emulator that you installed, install Git. For example, in Termux, enter `apt install git` and then type `y` when prompted. 

{% endnote %}

```

**Review by skedwards88 — COMMENTED** [review]
Thanks for adding this info! I made a suggestion to match the info with the GitHub docs style. I'll go ahead an incorporate that. Our repo is currently frozen until early January, but we will merge this for you once the freeze lifts.

**Review by skedwards88 — APPROVED** [review]

**Pull request merged**
